gamemaker.py

```diff
@@ -30,4 +32,52 @@ def logout(self):
         return return_value
 
     def display_menu(self):
-        return "Options\n\nlog out\ndisplay status\n"
+        return "Options\n\nmake team\n\nlog out\ndisplay status\n"
```

Comment:
Should display something like this:

Options

maketeam [new team name] [new team password]